helmclient.go

```diff
@@ -158,13 +209,30 @@ func (c *Client) GetReleaseHistory(releaseName string) (*ReleaseHistory, error)
 
 // InstallFromTarball installs a chart packaged in the given tarball.
 func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.InstallOption) error {
-	t, err := c.newTunnel()
-	if err != nil {
-		return microerror.Mask(err)
+	o := func() error {
+		t, err := c.newTunnel()
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		defer c.closeTunnel(t)
+
+		_, err = c.newHelmClientFromTunnel(t).InstallRelease(path, ns, options...)
+		if IsReleaseNotFound(err) {
+			return backoff.Permanent(releaseNotFoundError)
```

Comment:
Would additional context information of release name help in this error and also one below?

Reply:
Done.